x-pack/plugins/grokdebugger/public/sections/grokdebugger/components/apply_editor_options.js

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Could you move this file under https://github.com/elastic/kibana/tree/master/x-pack/plugins/grokdebugger/public/lib so all that's left in the `components` folder (where this file currently resides) are the actual components themselves.

Reply:
I wondered about this when I put it there but I figured if there was a problem you'd mention it 👍